package.json

```diff
@@ -6,10 +6,11 @@
   "main": "dist/mutation-test-elements.js",
   "scripts": {
     "clean": "rimraf packages/*/{dist,dist-test}",
-    "all": "npm run lint && npm run build && npm run test",
+    "all": "npm run clean && npm run lint && npm run build && npm run test",
     "postinstall": "lerna bootstrap && link-parent-bin",
-    "lint": "tslint -p tsconfig.lint.json",
-    "prebuild": "npm run clean",
+    "lint": "npm run lint:json && npm run lint:ts",
+    "lint:ts": "eslint --ext .js,.ts --ignore-path .gitignore .",
+    "lint:json": "prettier --check .eslintrc *.json",
```

Comment:
Can we also lint scss files?

Reply:
Yes, let's add that in a separate PR. This one is already kind of big